src/screens/UScreenOptionsWebcam.pas

```diff
@@ -193,7 +202,7 @@ constructor TScreenOptionsWebcam.Create;
 
   Theme.OptionsWebcam.SelectWebcam.showArrows := true;
   Theme.OptionsWebcam.SelectWebcam.oneItemOnly := true;
-  ID := AddSelectSlide(Theme.OptionsWebcam.SelectWebcam, Ini.WebCamID, WebcamsIDs);
+  WCID := AddSelectSlide(Theme.OptionsWebcam.SelectWebcam, Ini.WebCamID, WebcamsIDs);
```

Comment:
please use something actually usefull like WebcamID or similar.

Reply:
Agree.

Reply:
You do realize that some of these commits are already part of the master branch since #343 got merged?

Reply:
@s09bQ5 To be honest, no. When I merged USDX/master into my fork to include the helpsystem-v2 into my fork, I had many merge conflicts and might have overlooked something. Is there anything I need/can do?